build/push-ci-build.sh

```diff
@@ -32,5 +32,11 @@ KUBE_GCS_LATEST_CONTENTS=${LATEST}
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "$KUBE_ROOT/build/common.sh"
 
-kube::release::gcs::release
-kube::release::gcs::publish_latest
+MAX_ATTEMPTS=3
+attempt=0
+while [[ ${attempt} -lt ${MAX_ATTEMPTS} ]]; do
+  kube::release::gcs::release && kube::release::gcs::publish_latest && break || true
```

Comment:
Nutty. Should we bother coding around it then or not? I've been thinking that we need an env variable for that eventually, but "yes |" is adequate as well if we just wanted to force it.